Packs/PhishingURL/ReleaseNotes/1_1_17.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### DBotPredictURLPhishing
+
+- Fixed an issue in which an error was raised when the `rasterize` command returned multiple responses for a single URL.
```

Comment:
note the addition of the new param and how it works

Reply:
That was added in the RN in `rasterize`